amgut/db/patches/0018.sql

```diff
@@ -0,0 +1,8 @@
+-- August 24, 2015
+-- add country column to zipcode table
+ALTER TABLE ag.zipcodes ADD COLUMN country varchar NOT NULL DEFAULT '';
+ALTER TABLE ag.zipcodes ALTER COLUMN country DROP DEFAULT;
+UPDATE ag.zipcodes SET country = 'United States' WHERE state IN ('AL','AK','AZ','AR','CA','CO','CT','DC','DE','FL','GA','HI','ID','IL','IN','IA','KS','KY','LA','ME','MD','MA','MI','MN','MS','MO','MT','NE','NV','NH','NJ','NM','NY','NC','ND','OH','OK','OR','PA','RI','SC','SD','TN','TX','UT','VT','VA','WA','WV','WI','WY');
```

Comment:
OK, after talking with Gail, this is how the countries stuff shakes out:
1) All information stored in the database is in ISO 3166 format, so we can geocode from google with ease. 
2) On pulldown, a new column in the country table, `EBI`, will be used to convert the country in the DB to the country EBI expects.